### PR TITLE
Python 3 fixes for the opening hours validator

### PR DESCRIPTION
**Pull request by fgouget**
oh_sanitizer: Fix some test failures that happened only in Python 3

### CODE REVIEW

File: modules/oh_sanitizer/main.py [hunk 1]
```diff
@@ -269,7 +271,7 @@ def time(self, args):
     
     def hms(self, args):
         combined = int(args[0])
-        h = combined / 100
+        h = int(combined / 100)
```
Comment (frodrigo) [inline modules/oh_sanitizer/main.py]: Just use integer div , replace "/" by "//"

Comment (fgouget) [reply modules/oh_sanitizer/main.py]:
```suggestion
        h = combined // 100
```

**Comment by fgouget** [comment]
I used Github's suggestion option to hopefully modify the commit to use an integer division.
This means losing compatibility with some old Python 2 versions but I guess that's ok.

**Comment by frodrigo** [comment]
The min pyhton 2 version of Osmose is 2.7, And it's work.

The plan is to do changes in phton2.7 compatible python3.

**Comment by frodrigo** [comment]
Thank

**Pull request merged**